src/Microsoft.Health.Dicom.Core/Features/Validation/UidValidation.cs

```diff
@@ -1,4 +1,4 @@
-﻿// -------------------------------------------------------------------------------------------------
```

Comment:
Revert?

Reply:
git is not allowing to rollback this to previous commit.